telp-dashboard-v1/app/lib/definitions.ts

```diff
@@ -3,7 +3,7 @@
 // For simplicity of teaching, we're manually defining these types.
 // However, these types are generated automatically if you're using an ORM such as Prisma.
 export type Users = {
-  id: number;
+  id: string;
```

Comment:
we may have to update other portions of our code which expected id as int. e.g. the data.ts file expected id as number not string